helm/prometheus-chart/values.yaml

```
@@ -868,7 +868,7 @@ grafana:
   affinity: {}
 
   adminUser: admin
-  # adminPassword: strongpassword
+  adminPassword: Ericom123$
```

Errr what?

---

```suggestion
  adminPassword: strongpassword
```
It is a suggestion message, so let's people decide which one use